pkg/graph/graph.go

```diff
@@ -80,24 +82,29 @@ func (graphs ConfigGraphPerEnvironment) SortConfigs(environment string) ([]confi
 	}
 	sortedCfgs := make([]config.Config, len(sortedNodes))
 	for i, n := range sortedNodes {
-		sortedCfgs[i] = *n.(configNode).config
+		sortedCfgs[i] = *n.(ConfigNode).Config
 	}
 
 	return sortedCfgs, nil
 }
 
+type SortedComponent struct {
```

Comment:
Write documentation on why the graph is not enough (it would be) but we also have the topology-sorted nodes in